docs/install/assets/serving.md

```diff
@@ -0,0 +1,13 @@
+# Knative Serving installation files (0.21 releases)
+
+| File name | Description | Dependencies|
+| --- | --- | --- |
+| serving-core.yaml | Knative Serving core components. | serving-crds.yaml |
+| serving-crds.yaml | Knative Serving core CRDs. | none |
+| serving-default-domain.yaml | Configuring Knative Serving to use <a href="http://xip.io">xip.io</a> as the default DNS suffix. | serving-core.yaml |
+| serving-domainmapping-crds.yaml | CRDs required by the DomainMapping feature. | none |
```

Comment:
| serving-domainmapping-crds.yaml | CRDs required by the Domain Mapping feature. | none |